Makefile

```diff
@@ -260,7 +260,8 @@ Rpack: clean_all
 	cp ./LICENSE xgboost
 	cat R-package/src/Makevars.in|sed '2s/.*/PKGROOT=./' | sed '3s/.*/ENABLE_STD_THREAD=0/' > xgboost/src/Makevars.in
 	cp xgboost/src/Makevars.in xgboost/src/Makevars.win
-	sed -i -e 's/@OPENMP_CXXFLAGS@/$$\(SHLIB_OPENMP_CFLAGS\)/g' xgboost/src/Makevars.win
+	sed -i -e 's/@OPENMP_CXXFLAGS@/$$\(SHLIB_OPENMP_CXXFLAGS\)/g' xgboost/src/Makevars.win
+	sed -i -e 's/-pthread/$$\(SHLIB_PTHREAD_FLAGS\)/g' xgboost/src/Makevars.win
```

Comment:
Do these two lines have any effect on Makevars.win? It contains neither `@OPENMP_CXXFLAGS@` nor `-pthread`.

Reply:
Note when submitted to CRAN we have `cp xgboost/src/Makevars.in xgboost/src/Makevars.win`. Therefore it actually works on the `Makevars.in` file in github.

Reply:
Got it. Thanks.